spa/src/index.ts

```diff
@@ -78,33 +76,27 @@ function preprocessActivator(
  * SPA.
  */
 function setupApps(modules: Array<[string, System.Module]>) {
-  const extensions = createExtensions();
-
   for (const [appName, appExports] of modules) {
     const setup = appExports.setupOpenMRS;
 
     if (typeof setup === "function") {
       const result = setup();
 
       if (result && typeof result === "object") {
-        System.import(singleSpa).then(({ registerApplication }) => {
-          const availableExtensions: Array<ExtensionDefinition> =
-            result.extensions ?? [];
+        System.import("single-spa").then(({ registerApplication }) => {
+          System.import("@openmrs/esm-extension-manager").then(({ registerExtension }) => {
+            const availableExtensions = result.extensions ?? [];
```

Comment:
Should we keep the type annotations for `availableExtensions`? `Array<ExtensionDefinition>`

Reply:
Yeah actually, now that I think of it, we can import that type statically. I'll add that back in.